pgmpy/models/MarkovChain.py

```diff
@@ -177,7 +177,7 @@ def add_transition_model(self, variable, transition_model):
         -----------
         variable: any hashable python object
             must be an existing variable of the model.
-        transition_model: dict
+        transition_model: dict or ndarray
```

Comment:
@kris-singh  Add a more detailed description for the parameter explaining what kind of matrix or dict is expected.